Classes/Repository/RepositoryCodeBlobViewController.swift

```diff
@@ -17,6 +17,16 @@ final class RepositoryCodeBlobViewController: UIViewController {
     private let textView = UITextView()
     private let feedRefresh = FeedRefresh()
     private let emptyView = EmptyView()
+    private var sharingPayload: Any?
```

Comment:
Why `Any?`?

Reply:
Because the payload (`activityItems`) for the `UIActivityViewController` accepts `Any`; in the future this may not be `String` but eg. `UIImage` too.